src/main/java/com/minecolonies/coremod/event/FMLEventHandler.java

```diff
@@ -51,6 +54,14 @@ public static void onPlayerLogin(@NotNull final PlayerEvent.PlayerLoggedInEvent
         }
     }
 
+    @SubscribeEvent
+    public static void onServerAboutToStart(FMLServerAboutToStartEvent event)
+    {
+        IReloadableResourceManager resourceManager = event.getServer().getResourceManager();
+
+        resourceManager.addReloadListener(new CrafterRecipeListener());
```

Comment:
can be straight getResourceManager().add for now I guess

Reply:
It can, I was setting up for future types.